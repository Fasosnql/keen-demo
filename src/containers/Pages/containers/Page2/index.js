import React, { Component } from 'react';
import KeenTrack from 'components/KeenTrack';

class Page2 extends Component {
  render() {
    return (
      <div>
        page2
      </div>
    );
  }
}

export default KeenTrack({
  item: 'page2'
})(Page2);
