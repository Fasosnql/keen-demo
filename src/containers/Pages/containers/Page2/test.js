import React from 'react';
import Page2 from './index';
import renderer from 'react-test-renderer';

it('page2 component renders correctly', () => {
  const tree = renderer
    .create(<Page2 />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});