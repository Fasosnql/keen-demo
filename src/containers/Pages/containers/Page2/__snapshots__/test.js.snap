// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page2 component renders correctly 1`] = `
<div>
  page2
</div>
`;
